Duplicati/Library/Common/IO/SystemIOWindows.cs

```diff
@@ -547,16 +547,15 @@ public string PathCombine(params string[] paths)
 
             return combinedPath;
         }
-
+        
         public void CreateSymlink(string symlinkfile, string target, bool asDir)
         {
             if (FileExists(symlinkfile) || DirectoryExists(symlinkfile))
-                throw new System.IO.IOException(string.Format("File already exists: {0}", symlinkfile));
+                throw new System.IO.IOException($"File already exists: {symlinkfile}");
 
             Alphaleonis.Win32.Filesystem.File.CreateSymbolicLink(PrefixWithUNC(symlinkfile),
-                                                                 target,
-                                                                 asDir ? Alphaleonis.Win32.Filesystem.SymbolicLinkTarget.Directory : Alphaleonis.Win32.Filesystem.SymbolicLinkTarget.File,
-                                                                 AlphaFS.PathFormat.LongFullPath);
+                target,
+                AlphaFS.PathFormat.LongFullPath);

```

Comment:
Is this change because the old method overload has been deprecated?

http://alphafs.alphaleonis.com/doc/2.2/api/html/8863EE02.htm

Reply:
Correct because it is deprecated, but it isn't related to this any way.

Let's close this PR once we're done discussing 4.6.2

I have have all the 4.6.2 changes in a new branch. I can put it into a new PR if we get the green light.